src/lib/transaction_snark/transaction_snark.ml

```diff
@@ -4781,12 +4790,11 @@ module Make_str (A : Wire_types.Concrete) = struct
             (Control.Poly.Proof (Lazy.force Mina_base.Proof.blockchain_dummy))
       in
       let account_update_digest_with_selected_chain =
-        Zkapp_command.Digest.Account_update.create ~signature_kind
-          account_update_with_dummy_auth
+        Zkapp_command.Digest.Account_update.create
+          ~signature_kind:other_signature_kind account_update_with_dummy_auth
```

Comment:
```suggestion
          ~signature_kind:(Other_network "dummy") account_update_with_dummy_auth
```

Reply:
The same `invalid` is always used, and this is only needed to ensure that dummy and testnet signature kinds aren't mixed, to strengthen test code reliability

Reply:
`Zkapp_command_logic.Make` is contraining the signatures and that functor is applied to other places. So I'd rather not touch here